2.1/README.md

```diff
@@ -0,0 +1,45 @@
+HTTP 301 Moved Permanently
+--------------------------
+
+**Where are the 2.1 aspnet-docker images?**
+
+In 2.1, the ASP.NET Core Docker images have migrated to https://github.com/dotnet/dotnet-docker.
+
+**How can I upgrade from 1.x/2.0 to 2.1?**
```

Comment:
I'm concerned that there is no mention of 2.1 still being in a preview state.  If a reader isn't aware of the 2.1 status, this can give the false impression that upgrading from pre 2.1 to 2.1 is fine and dandy.

Reply:
I had the same concern about making the `2.1-sdk` tag before RTM...but that ship sailed.

I can add a note about 2.1 being preview for now. I'm hoping that note is also prominent enough in the dotnet-docker README.